app/routes/task_routes.py

```diff
@@ -1 +1,87 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response, make_response
+from app.models.task import Task
+from .route_utilities import validate_model, create_model, get_models_with_filters
```

Comment:
Remove unused imports
```suggestion
from .route_utilities import validate_model, create_model
```

Reply:
Why? I need this helper function.
Refactored as requested.